src/actor/ActorRealm.ts

```diff
@@ -0,0 +1,27 @@
+import { actorMixin, ValidActorMessageName } from "./Actor.js";
+
+export class ActorIDBRealm {
+  private readonly actors = new Map<ValidActorMessageName, actorMixin<any>>();
+
+  addActor<ActorName extends ValidActorMessageName>(
+    actorName: ActorName,
+    actor: actorMixin<ActorMessageType[ActorName]>
+  ) {
+    this.actors.set(actorName, actor);
```

Comment:
Let's throw if we already have this actor.

Reply:
Done